source/text/labelgeometry.ts

```diff
@@ -79,21 +115,23 @@ export class LabelGeometry extends Geometry {
 
         /* Please note the implicit bind in attribEnable */
 
-        /* quadVertex */
-        this._buffers[0].attribEnable(indices[0], 2, gl.FLOAT, false, 0, 0, true, false);
-        gl2facade.vertexAttribDivisor(indices[0], 0);
-        /* texCoords */
-        this._buffers[1].attribEnable(indices[1], 4, gl.FLOAT, false, 4 * 4, 0, true, false);
-        gl2facade.vertexAttribDivisor(indices[1], 1);
-        /* origin */
-        this._buffers[2].attribEnable(indices[2], 3, gl.FLOAT, false, 3 * 4, 0, true, false);
-        gl2facade.vertexAttribDivisor(indices[2], 1);
-        /* tangent */
-        this._buffers[3].attribEnable(indices[3], 3, gl.FLOAT, false, 3 * 4, 0, true, false);
-        gl2facade.vertexAttribDivisor(indices[3], 1);
-        /* up */
-        this._buffers[4].attribEnable(indices[4], 3, gl.FLOAT, false, 3 * 4, 0, true, false);
-        gl2facade.vertexAttribDivisor(indices[4], 1);
+        this._vertices.attribEnable(this._vertexLocation,
+            2, gl.FLOAT, false, 8, 0, true, false);
+        gl2facade.vertexAttribDivisor(this._vertexLocation, 0);
+        this._texCoords.attribEnable(this._texCoordLocation,
+            4, gl.FLOAT, false, 16, 0, true, false);
+        gl2facade.vertexAttribDivisor(this._texCoordLocation, 1);
+
+        this._origins.attribEnable(this._originLocation,
+            3, gl.FLOAT, false, 12, 0, true, false);
+        gl2facade.vertexAttribDivisor(this._originLocation, 1);
+
+        this._tangents.attribEnable(this._tangentLocation,
+            3, gl.FLOAT, false, 12, 0, true, false);
+        gl2facade.vertexAttribDivisor(this._tangentLocation, 1);
+        this._bitangents.attribEnable(this._bitangentLocation,
+            3, gl.FLOAT, false, 12, 0, true, false);
+        gl2facade.vertexAttribDivisor(this._bitangentLocation, 1);
```

Comment:
Is there a meaning behind the grouping (missing empty lines)? Same for `unbindBuffers()`.